src/gazebo_gimbal_controller_plugin.cpp

```diff
@@ -28,7 +28,7 @@ GZ_REGISTER_MODEL_PLUGIN(GimbalControllerPlugin)
 GimbalControllerPlugin::GimbalControllerPlugin()
   :status("closed")
 {
-  /// TODO: make these gains part of sdf xml
+  /// defaults if sdf xml doesn't contain any pid gains
```

Comment:
@EliaTarasov can you please set constants on the `gazebo_gimbal_controller_plugin.h` to set the PID values here? Similar to what is done with other constants? Like `kPIDPitchP`, `kPIDPitchI`, etc.?

Reply:
Yes, it's done